Extensions/Video/videoruntimeobject.js

```diff
@@ -270,10 +303,9 @@ gdjs.VideoRuntimeObject.prototype.getPlaybackSpeed = function() {
  * TODO: Investigate how to dispose the video source?
  *
  * @private
+ * @param {gdjs.RuntimeScene} runtimeScene The {@link gdjs.RuntimeScene} this objects has to be removed from.
```

Comment:
This callback is independent from the notion of object:

`The {@link gdjs.RuntimeScene} that is being unloaded`

Reply:
fixed